routes/reviews.js

```diff
@@ -96,4 +96,30 @@ router.post('/', (req, res) => {
     });
 });
 
+
+
+router.delete('/', (req, res) => {
+   
+
+    //The film's ID of the query is saved in ReviewId
+    var ReviewId=req.params.imbId;
+
+    //Now we execute the delete operation
+    reviews.deleteOne({imdbId:ReviewId}, (err)=>{
+        
+        if(!err){ //if the imdbId exists, the review will be deleted
+
+            console.log("The review has been deleted")
+            return res.sendStatus(201);
+
+        }else{ //if the imdbId does not exist, an error message will be sent.
+
+            console.log("The review doesn't exist");
+            return res.sendStatus(400);
```

Comment:
When not found, it should return a 404 code. I think this is because of my mistake on the Swagger documentation.

Reply:
I will update the swagger API doc and get back to this later.